README.md

```diff
@@ -203,16 +202,16 @@ class EquatableDateTime extends DateTime
   ]) : super(year, month, day, hour, minute, second, millisecond, microsecond);
 
   @override
-  List get props {
+  List<Object> get props {
```

Comment:
Does the linter complain if this is changed to `List<dynamic>` ?

Reply:
I don't think it would but it in my opinion it's okay to use type `Object` instead.